docs/section21/Rule21-3.md

```diff
@@ -0,0 +1,44 @@
+
+# Boiler - Rule 21-3  
+
+**Rule ID:** 21-3  
+**Rule Description:** Plant capacities shall be based on coincident loads.  
+**Rule Assertion:** B-RMR FluidLoop.heating_design_and_control: is_sized_using_coincident_load = True  
+**Appendix G Section:** Section 21 Boiler  
+**Appendix G Section Reference:** Section G3.1.2.2 Building System-Specific Modeling Requirements for the Baseline model  
+
+**Applicability:** All required data elements exist for B-RMR  
+**Applicability Checks:**  
+
+1. B-RMR is modeled with heating fluid loop(s)
+
+**Manual Check:** None  
+**Evaluation Context:** Building  
+**Data Lookup:** None  
+**Function Call:** None  
+
+## Rule Logic:  
+
+- For each fluid loop in B_RMR: `for fluid_loop_b in B_RMR.ASHRAE229.fluid_loops:`
+
```

Comment:
Seems like we also need to have a check that there is no more than 1 HW loop in the baseline. I don't think there is a rule like that now.

Reply:
Rule is now added to check that no more than 1 HW loop is modeled in the baseline, Rule 21-18.